README.md

```diff
@@ -1 +1,94 @@
-# kaiju-plugin-utils
\ No newline at end of file
+# kaiju-plugin-utils
+
+Kaiju plugin utils is a helpful library of common optional utilities used when creating plugins for Kaiju.
+
+## Development notes
+This module uses app-root-path to determine root path to the kaiju node server. This library determines the root path by walking up the current directory path to be one level below the last occurrence of 'node_modules'. This works great normally but can be incorrect when this module is setup as a file dependency because the directory is symlinked instead of copied. For this module to work as expected in development you cannot require this module as a file.
+
+## Plugin Utils
+Common optional utilities to be used when developing kaiju plugins. There is a dependency on webpack for preview creation. If you don't need webpack, you might not want to use these utils.
+
+### Usage
+```js
+const { PluginUtils } = require('kaiju-plugin-utils');
+
+PluginUtils.rootPath() // for example
+```
+
+### runCompiler
+Runs the webpack compiler on the files included in the memory file system with the config supplied.
+
+**Arguments:**
+Memory File System - A memory file system containing the files to be webpacked.
+
+Config - The webpack config to be used by the compiler.
+
+**Return values:** Compiler file system - A promise for a memory file system contained the output from the webpack compiler.
```

Comment:
The description might be need to be updated. Might read cleaner swapping `contained`  to `containing` in `memory file system contained`